README.md

```diff
@@ -130,6 +131,15 @@ Object containing the following attributes:
 | `name` | `string` | filename of the rendition to create |
 | `path` | `string` | Absolute path to store rendition locally (must put rendition here in order to be uploaded to cloud storage) |
 | `index` | `number` | number used to identify a rendition |
+| `watermark` | `object` | See [watermark](#watermark)
```

Comment:
This isn't part of `rendition` in the JS API. It would be part of `instructions`.

We do not have a section on post processing options in this README yet. Only a pointer to https://docs.adobe.com/content/help/en/asset-compute/using/api.html#rendition-instructions which we need to update instead. It would be nice to have a list here as well but then we have two places to update which easily goes out of sync.